src/middleware.ts

```diff
@@ -1,25 +1,32 @@
 
 import { NextResponse } from 'next/server';
 import type { NextRequest } from 'next/server';
-
+// export let UserIsLogin = false ; 
 export function middleware(request: NextRequest) {
-    const protectedPaths = ['/home', '/'];
+    const protectedPaths = ['/home', '/' , '/projects',];
 
     const authToken = request.cookies.get('authToken')?.value;
-    // console.log(authToken);
+    
+    // if(authToken){
+    //     UserIsLogin = true
+    // }
+    // console.log(authToken , UserIsLogin);
 
     const isProtectedPath = protectedPaths.some((path) =>
         request.nextUrl.pathname.startsWith(path)
```

Comment:
I think using `request.nextUrl.pathname.includes(path)` is better than startsWith?
Do you agree with that? and why?